hub/util/exceptions.py

```diff
@@ -309,12 +309,23 @@ def __init__(self, htype: str, available_htypes: Sequence[str]):
 class TensorMetaInvalidHtypeOverwriteValue(MetaError):
     def __init__(self, key: str, value: Any, explanation: str = ""):
         super().__init__(
-            "Invalid value {} for tensor meta key {}. {}".format(
+            "Invalid value '{}' for tensor meta key {}. {}".format(
```

Comment:
let's stick to fstrings everywhere so we don't need to do fstring replacement PRs in the future.

Reply:
was an upstream change i just pulled in